vllm/compilation/levels.py

```diff
@@ -0,0 +1,9 @@
+# constants for the levels of the compilation process
+
+
```

Comment:
I think it would be good to add a more detailed description of each level here.

Reply:
I will add a doc page and refer to the doc page later. docstring in code is limited here for expressiveness.